client/resource_group/controller/controller.go

```diff
@@ -1044,6 +1056,22 @@ func (gc *groupCostController) collectRequestAndConsumption(selectTyp selectType
 	return req
 }
 
+// If the resource group has background jobs, we should not record consumption and wait for it.
+func (gc *groupCostController) isBackgroundRequest(requestResource string) bool {
+	gc.metaLock.Lock()
+	defer gc.metaLock.Unlock()
+
+	if gc.meta.BackgroundSettings != nil {
+		jobTypes := gc.meta.BackgroundSettings.JobTypes
```

Comment:
I think we can cache  job type, it's changed with low frequency, so we can make a quick path to check it.